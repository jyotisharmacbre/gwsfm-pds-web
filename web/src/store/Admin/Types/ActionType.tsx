export enum ActionType {
	GET_PROJECT_PARAMETERS_SUCCESS = 'GET_PROJECT_PARAMETERS_SUCCESS',
	GET_PROJECT_PARAMETERS_ERROR = 'GET_PROJECT_PARAMETERS_ERROR'
}
