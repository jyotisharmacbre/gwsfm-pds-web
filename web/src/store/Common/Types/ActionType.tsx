export enum ActionType {
  DEFAULT = 'DEFAULT'
}
