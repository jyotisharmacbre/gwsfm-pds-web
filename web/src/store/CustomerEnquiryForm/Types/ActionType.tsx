export enum ActionType {
  DEFAULT = 'DEFAULT',
  PROJECT_ADD = 'PROJECT_ADD',
  PROJECT_ADD_ERROR = 'PROJECT_ADD_ERROR',
  PROJECT_EDIT_SUCCESS = 'PROJECT_EDIT_SUCCESS',
  GET_ENQUIRY_OVERVIEW_SUCCESS = 'GET_ENQUIRY_OVERVIEW_SUCCESS',
  GET_ENQUIRY_OVERVIEW_ERROR = 'GET_ENQUIRY_OVERVIEW_ERROR',
  GET_PROJECT_DETAIL_SUCCESS = 'GET_PROJECT_DETAIL_SUCCESS',
  GET_PROJECT_DETAIL_ERROR = 'GET_PROJECT_DETAIL_ERROR',
  RESET_PROJECT_DETAIL_STATE = 'RESET_PROJECT_DETAIL_STATE',
  SET_PROJECT_ID_STATE = 'SET_PROJECT_ID_STATE'
}
