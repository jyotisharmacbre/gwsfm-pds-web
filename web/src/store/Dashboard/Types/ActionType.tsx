export enum ActionType {
  PROJECT_DASHBOARD_GRID_ERROR = 'PROJECT_DASHBOARD_GRID_ERROR',
  PROJECT_DASHBOARD_GRID_DETAILS = 'PROJECT_DASHBOARD_GRID_DETAILS'
}
