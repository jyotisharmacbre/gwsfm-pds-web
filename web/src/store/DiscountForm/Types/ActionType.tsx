export enum ActionType {
	DISCOUNT_FORM_DATA_ADD = 'DISCOUNT_FORM_DATA_ADD',
	DISCOUNT_FORM_DATA_ERROR = 'DISCOUNT_FORM_DATA_ERROR',
	DISCOUNT_FORM_DATA_EDIT = 'DISCOUNT_FORM_DATA_EDIT',
	RESET_DISCOUNT_FORM_STATE = 'RESET_DISCOUNT_FORM_STATE',
	RESET_DISCOUNT_FORM_STATE_NOTIFIER = 'RESET_DISCOUNT_FORM_STATE_NOTIFIER',
	DISCOUNT_FORM_DATA_GET = 'DISCOUNT_FORM_DATA_GET',
	DISCOUNT_FORM_DATA_GET_ERROR = 'DISCOUNT_FORM_DATA_GET_ERROR'
}
