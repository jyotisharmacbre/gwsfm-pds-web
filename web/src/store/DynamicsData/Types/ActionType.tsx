export enum ActionType {
  DYNAMIC_CONTRACT_DATA_GET_SUCCESS = 'DYNAMIC_CONTRACT_DATA_GET_SUCCESS',
  DYNAMIC_CONTRACT_DATA_GET_ERROR = 'DYNAMIC_CONTRACT_DATA_GET_ERROR',
  DYNAMIC_COMPANY_DATA_GET_SUCCESS = 'DYNAMIC_COMPANY_DATA_GET_SUCCESS',
  DYNAMIC_COMPANY_DATA_GET_ERROR = 'DYNAMIC_COMPANY_DATA_GET_ERROR',
  DYNAMIC_OTHER_CONTRACT_SUCCESS = 'DYNAMIC_OTHER_CONTRACT_SUCCESS',
  DYNAMIC_OTHER_COMPANY_SUCCESS = 'DYNAMIC_OTHER_COMPANY_SUCCESS',
  DYNAMIC_SUB_CONTRACTOR_DATA_GET_SUCCESS = 'DYNAMIC_SUB_CONTRACTOR_DATA_GET_SUCCESS',
  DYNAMIC_SUB_CONTRACTOR_DATA_GET_ERROR = 'DYNAMIC_SUB_CONTRACTOR_DATA_GET_ERROR',
  DYNAMIC_OTHER_SUB_CONTRACTOR_SUCCESS = 'DYNAMIC_OTHER_SUB_CONTRACTOR_SUCCESS',
}
