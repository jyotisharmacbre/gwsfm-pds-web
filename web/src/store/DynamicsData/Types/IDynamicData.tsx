export interface IDynamicContractData {
  id: string;
  name: string;
}

export interface IDynamicCompanyData {
  id: string;
  name: string;
}

export interface IDynamicSubContractorData {
  id: string;
  name: string;
}

export interface IDynamicOther {
  id: string;
  label: string;
}
