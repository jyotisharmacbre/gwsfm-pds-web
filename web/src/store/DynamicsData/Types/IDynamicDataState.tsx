import {
  IDynamicCompanyData,
  IDynamicContractCustomerData,
  IDynamicOther,
  IDynamicSubContractorData,
  IDynamicsDivision,
  IDynamicBusinessUnits
} from './IDynamicData';

export interface IDynamicDataState {
  dynamicsCompany: Array<IDynamicCompanyData>;
  dynamicsListOfDivision: Array<IDynamicsDivision>;
  dynamicsContract: Array<IDynamicContractCustomerData>;
  error: string | null;
  dynamicsOtherContract: Array<IDynamicOther>;
  dynamicsOtherCompany: Array<IDynamicOther>;
  dynamicsSubcontractor: Array<IDynamicSubContractorData>;
  dynamicsOtherSubContractor: Array<IDynamicOther>;
  dynamicsListOfBusinessUnits:Array<IDynamicBusinessUnits>;
}
