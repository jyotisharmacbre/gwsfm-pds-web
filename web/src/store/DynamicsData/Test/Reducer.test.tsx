import { ActionType } from './../Types/ActionType';
import dynamicDataReducer, { initialState } from '../Reducer';

describe('Dynamic data reducer test cases', ()=>{

it('should handle DYNAMIC_CONTRACT_DATA_GET_SUCCESS ', () => {
    const DYNAMIC_CONTRACT_DATA_GET_SUCCESS: any = {
        type: ActionType.DYNAMIC_CONTRACT_DATA_GET_SUCCESS,
        payload: initialState
    };
    expect(dynamicDataReducer(initialState, DYNAMIC_CONTRACT_DATA_GET_SUCCESS)).toMatchSnapshot();
});

it('should handle DYNAMIC_CONTRACT_DATA_GET_SUCCESS ', () => {
    const DYNAMIC_CONTRACT_DATA_GET_ERROR: any = {
        type: ActionType.DYNAMIC_CONTRACT_DATA_GET_ERROR,
        payload: {error: true}
    };
    expect(dynamicDataReducer(initialState, DYNAMIC_CONTRACT_DATA_GET_ERROR)).toMatchSnapshot();
});

it('should handle DYNAMIC_CONTRACT_DATA_GET_SUCCESS ', () => {
    const DYNAMIC_COMPANY_DATA_GET_SUCCESS: any = {
        type: ActionType.DYNAMIC_COMPANY_DATA_GET_SUCCESS,
        payload: initialState
    };
    expect(dynamicDataReducer(initialState, DYNAMIC_COMPANY_DATA_GET_SUCCESS)).toMatchSnapshot();
});

it('should handle DYNAMIC_COMPANY_DATA_GET_ERROR ', () => {
    const DYNAMIC_COMPANY_DATA_GET_ERROR: any = {
        type: ActionType.DYNAMIC_COMPANY_DATA_GET_ERROR,
        payload: {error: true}
    };
    expect(dynamicDataReducer(initialState, DYNAMIC_COMPANY_DATA_GET_ERROR)).toMatchSnapshot();
});

it('should handle DYNAMIC_OTHER_CONTRACT_SUCCESS ', () => {
    const DYNAMIC_OTHER_CONTRACT_SUCCESS: any = {
        type: ActionType.DYNAMIC_OTHER_CONTRACT_SUCCESS,
        payload: initialState
    };
    expect(dynamicDataReducer(initialState, DYNAMIC_OTHER_CONTRACT_SUCCESS)).toMatchSnapshot();
});

it('should handle DYNAMIC_OTHER_COMPANY_SUCCESS ', () => {
    const DYNAMIC_OTHER_COMPANY_SUCCESS: any = {
        type: ActionType.DYNAMIC_OTHER_COMPANY_SUCCESS,
        payload: initialState
    };
    expect(dynamicDataReducer(initialState, DYNAMIC_OTHER_COMPANY_SUCCESS)).toMatchSnapshot();
});

it('should handle DYNAMIC_SUB_CONTRACTOR_DATA_GET_SUCCESS ', () => {
    const DYNAMIC_SUB_CONTRACTOR_DATA_GET_SUCCESS: any = {
        type: ActionType.DYNAMIC_SUB_CONTRACTOR_DATA_GET_SUCCESS,
        payload: initialState
    };
    expect(dynamicDataReducer(initialState, DYNAMIC_SUB_CONTRACTOR_DATA_GET_SUCCESS)).toMatchSnapshot();
});

it('should handle DYNAMIC_SUB_CONTRACTOR_DATA_GET_ERROR ', () => {
    const DYNAMIC_SUB_CONTRACTOR_DATA_GET_ERROR: any = {
        type: ActionType.DYNAMIC_SUB_CONTRACTOR_DATA_GET_ERROR,
        payload: {error:true}
    };
    expect(dynamicDataReducer(initialState, DYNAMIC_SUB_CONTRACTOR_DATA_GET_ERROR)).toMatchSnapshot();
});

it('should handle DYNAMIC_OTHER_SUB_CONTRACTOR_SUCCESS ', () => {
    const DYNAMIC_OTHER_SUB_CONTRACTOR_SUCCESS: any = {
        type: ActionType.DYNAMIC_OTHER_SUB_CONTRACTOR_SUCCESS,
        payload: initialState
    };
    expect(dynamicDataReducer(initialState, DYNAMIC_OTHER_SUB_CONTRACTOR_SUCCESS)).toMatchSnapshot();
});

it('should handle DYNAMIC_DIVISION_DATA_GET_SUCCESS ', () => {
    const DYNAMIC_DIVISION_DATA_GET_SUCCESS: any = {
        type: ActionType.DYNAMIC_DIVISION_DATA_GET_SUCCESS,
        payload: initialState.dynamicsListOfDivision
    };
    expect(dynamicDataReducer(initialState, DYNAMIC_DIVISION_DATA_GET_SUCCESS)).toMatchSnapshot();
});

it('should handle DYNAMIC_DIVISION_DATA_GET_ERROR ', () => {
    const DYNAMIC_DIVISION_DATA_GET_ERROR: any = {
        type: ActionType.DYNAMIC_DIVISION_DATA_GET_ERROR,
        payload: {error: true}
    };
    expect(dynamicDataReducer(initialState, DYNAMIC_DIVISION_DATA_GET_ERROR)).toMatchSnapshot();
});

it('should handle DYNAMIC_BUSINESSUNIT_SUCCESS ', () => {
    const DYNAMIC_BUSINESSUNIT_SUCCESS: any = {
        type: ActionType.DYNAMIC_BUSINESSUNIT_SUCCESS,
        payload: initialState.dynamicsListOfBusinessUnits
    };
    expect(dynamicDataReducer(initialState, DYNAMIC_BUSINESSUNIT_SUCCESS)).toMatchSnapshot();
}); 

it('should handle DYNAMIC_BUSINESSUNIT_ERROR ', () => {
    const DYNAMIC_BUSINESSUNIT_ERROR: any = {
        type: ActionType.DYNAMIC_BUSINESSUNIT_ERROR,
        payload: {error: true}
    };
    expect(dynamicDataReducer(initialState, DYNAMIC_BUSINESSUNIT_ERROR)).toMatchSnapshot();
}); 

it('should handle DYNAMIC_CONTRACTLIST_SUCCESS ', () => {
    const DYNAMIC_CONTRACTLIST_SUCCESS: any = {
        type: ActionType.DYNAMIC_CONTRACTLIST_SUCCESS,
        payload: initialState
    };
    expect(dynamicDataReducer(initialState, DYNAMIC_CONTRACTLIST_SUCCESS)).toMatchSnapshot();
}); 

it('should handle DYNAMIC_CONTRACTLIST_ERROR ', () => {
    const DYNAMIC_CONTRACTLIST_ERROR: any = {
        type: ActionType.DYNAMIC_CONTRACTLIST_ERROR,
        payload: initialState
    };
    expect(dynamicDataReducer(initialState, DYNAMIC_CONTRACTLIST_ERROR)).toMatchSnapshot();
}); 

})