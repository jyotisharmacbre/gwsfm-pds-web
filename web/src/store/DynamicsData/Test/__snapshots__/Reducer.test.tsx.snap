// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dynamic data reducer test cases should handle DYNAMIC_BUSINESSUNIT_ERROR  1`] = `
Object {
  "dynamicsCompany": Array [],
  "dynamicsContract": Array [],
  "dynamicsListOfBusinessUnits": Array [],
  "dynamicsListOfDivision": Array [],
  "dynamicsOtherCompany": Array [],
  "dynamicsOtherContract": Array [],
  "dynamicsOtherSubContractor": Array [],
  "dynamicsSubcontractor": Array [],
  "error": Object {
    "error": true,
  },
}
`;

exports[`Dynamic data reducer test cases should handle DYNAMIC_BUSINESSUNIT_SUCCESS  1`] = `
Object {
  "dynamicsCompany": Array [],
  "dynamicsContract": Array [],
  "dynamicsListOfBusinessUnits": Object {
    "dynamicsCompany": Array [],
    "dynamicsContract": Array [],
    "dynamicsListOfBusinessUnits": Array [],
    "dynamicsListOfDivision": Array [],
    "dynamicsOtherCompany": Array [],
    "dynamicsOtherContract": Array [],
    "dynamicsOtherSubContractor": Array [],
    "dynamicsSubcontractor": Array [],
    "error": null,
  },
  "dynamicsListOfDivision": Array [],
  "dynamicsOtherCompany": Array [],
  "dynamicsOtherContract": Array [],
  "dynamicsOtherSubContractor": Array [],
  "dynamicsSubcontractor": Array [],
  "error": null,
}
`;

exports[`Dynamic data reducer test cases should handle DYNAMIC_COMPANY_DATA_GET_ERROR  1`] = `
Object {
  "dynamicsCompany": Array [],
  "dynamicsContract": Array [],
  "dynamicsListOfBusinessUnits": Array [],
  "dynamicsListOfDivision": Array [],
  "dynamicsOtherCompany": Array [],
  "dynamicsOtherContract": Array [],
  "dynamicsOtherSubContractor": Array [],
  "dynamicsSubcontractor": Array [],
  "error": Object {
    "error": true,
  },
}
`;

exports[`Dynamic data reducer test cases should handle DYNAMIC_CONTRACT_DATA_GET_SUCCESS  1`] = `
Object {
  "dynamicsCompany": Array [],
  "dynamicsContract": Object {
    "dynamicsCompany": Array [],
    "dynamicsContract": Array [],
    "dynamicsListOfBusinessUnits": Array [],
    "dynamicsListOfDivision": Array [],
    "dynamicsOtherCompany": Array [],
    "dynamicsOtherContract": Array [],
    "dynamicsOtherSubContractor": Array [],
    "dynamicsSubcontractor": Array [],
    "error": null,
  },
  "dynamicsListOfBusinessUnits": Array [],
  "dynamicsListOfDivision": Array [],
  "dynamicsOtherCompany": Array [],
  "dynamicsOtherContract": Array [],
  "dynamicsOtherSubContractor": Array [],
  "dynamicsSubcontractor": Array [],
  "error": null,
}
`;

exports[`Dynamic data reducer test cases should handle DYNAMIC_CONTRACT_DATA_GET_SUCCESS  2`] = `
Object {
  "dynamicsCompany": Array [],
  "dynamicsContract": Array [],
  "dynamicsListOfBusinessUnits": Array [],
  "dynamicsListOfDivision": Array [],
  "dynamicsOtherCompany": Array [],
  "dynamicsOtherContract": Array [],
  "dynamicsOtherSubContractor": Array [],
  "dynamicsSubcontractor": Array [],
  "error": Object {
    "error": true,
  },
}
`;

exports[`Dynamic data reducer test cases should handle DYNAMIC_CONTRACT_DATA_GET_SUCCESS  3`] = `
Object {
  "dynamicsCompany": Object {
    "dynamicsCompany": Array [],
    "dynamicsContract": Array [],
    "dynamicsListOfBusinessUnits": Array [],
    "dynamicsListOfDivision": Array [],
    "dynamicsOtherCompany": Array [],
    "dynamicsOtherContract": Array [],
    "dynamicsOtherSubContractor": Array [],
    "dynamicsSubcontractor": Array [],
    "error": null,
  },
  "dynamicsContract": Array [],
  "dynamicsListOfBusinessUnits": Array [],
  "dynamicsListOfDivision": Array [],
  "dynamicsOtherCompany": Array [],
  "dynamicsOtherContract": Array [],
  "dynamicsOtherSubContractor": Array [],
  "dynamicsSubcontractor": Array [],
  "error": null,
}
`;

exports[`Dynamic data reducer test cases should handle DYNAMIC_CONTRACTLIST_ERROR  1`] = `
Object {
  "dynamicsCompany": Array [],
  "dynamicsContract": Array [],
  "dynamicsListOfBusinessUnits": Array [],
  "dynamicsListOfDivision": Array [],
  "dynamicsOtherCompany": Array [],
  "dynamicsOtherContract": Array [],
  "dynamicsOtherSubContractor": Array [],
  "dynamicsSubcontractor": Array [],
  "error": Object {
    "dynamicsCompany": Array [],
    "dynamicsContract": Array [],
    "dynamicsListOfBusinessUnits": Array [],
    "dynamicsListOfDivision": Array [],
    "dynamicsOtherCompany": Array [],
    "dynamicsOtherContract": Array [],
    "dynamicsOtherSubContractor": Array [],
    "dynamicsSubcontractor": Array [],
    "error": null,
  },
}
`;

exports[`Dynamic data reducer test cases should handle DYNAMIC_CONTRACTLIST_SUCCESS  1`] = `
Object {
  "dynamicsCompany": Array [],
  "dynamicsContract": Object {
    "dynamicsCompany": Array [],
    "dynamicsContract": Array [],
    "dynamicsListOfBusinessUnits": Array [],
    "dynamicsListOfDivision": Array [],
    "dynamicsOtherCompany": Array [],
    "dynamicsOtherContract": Array [],
    "dynamicsOtherSubContractor": Array [],
    "dynamicsSubcontractor": Array [],
    "error": null,
  },
  "dynamicsListOfBusinessUnits": Array [],
  "dynamicsListOfDivision": Array [],
  "dynamicsOtherCompany": Array [],
  "dynamicsOtherContract": Array [],
  "dynamicsOtherSubContractor": Array [],
  "dynamicsSubcontractor": Array [],
  "error": null,
}
`;

exports[`Dynamic data reducer test cases should handle DYNAMIC_DIVISION_DATA_GET_ERROR  1`] = `
Object {
  "dynamicsCompany": Array [],
  "dynamicsContract": Array [],
  "dynamicsListOfBusinessUnits": Array [],
  "dynamicsListOfDivision": Array [],
  "dynamicsOtherCompany": Array [],
  "dynamicsOtherContract": Array [],
  "dynamicsOtherSubContractor": Array [],
  "dynamicsSubcontractor": Array [],
  "error": Object {
    "error": true,
  },
}
`;

exports[`Dynamic data reducer test cases should handle DYNAMIC_DIVISION_DATA_GET_SUCCESS  1`] = `
Object {
  "dynamicsCompany": Array [],
  "dynamicsContract": Array [],
  "dynamicsListOfBusinessUnits": Array [],
  "dynamicsListOfDivision": Object {
    "dynamicsCompany": Array [],
    "dynamicsContract": Array [],
    "dynamicsListOfBusinessUnits": Array [],
    "dynamicsListOfDivision": Array [],
    "dynamicsOtherCompany": Array [],
    "dynamicsOtherContract": Array [],
    "dynamicsOtherSubContractor": Array [],
    "dynamicsSubcontractor": Array [],
    "error": null,
  },
  "dynamicsOtherCompany": Array [],
  "dynamicsOtherContract": Array [],
  "dynamicsOtherSubContractor": Array [],
  "dynamicsSubcontractor": Array [],
  "error": null,
}
`;

exports[`Dynamic data reducer test cases should handle DYNAMIC_OTHER_COMPANY_SUCCESS  1`] = `
Object {
  "dynamicsCompany": Array [],
  "dynamicsContract": Array [],
  "dynamicsListOfBusinessUnits": Array [],
  "dynamicsListOfDivision": Array [],
  "dynamicsOtherCompany": Object {
    "dynamicsCompany": Array [],
    "dynamicsContract": Array [],
    "dynamicsListOfBusinessUnits": Array [],
    "dynamicsListOfDivision": Array [],
    "dynamicsOtherCompany": Array [],
    "dynamicsOtherContract": Array [],
    "dynamicsOtherSubContractor": Array [],
    "dynamicsSubcontractor": Array [],
    "error": null,
  },
  "dynamicsOtherContract": Array [],
  "dynamicsOtherSubContractor": Array [],
  "dynamicsSubcontractor": Array [],
  "error": null,
}
`;

exports[`Dynamic data reducer test cases should handle DYNAMIC_OTHER_CONTRACT_SUCCESS  1`] = `
Object {
  "dynamicsCompany": Array [],
  "dynamicsContract": Array [],
  "dynamicsListOfBusinessUnits": Array [],
  "dynamicsListOfDivision": Array [],
  "dynamicsOtherCompany": Array [],
  "dynamicsOtherContract": Object {
    "dynamicsCompany": Array [],
    "dynamicsContract": Array [],
    "dynamicsListOfBusinessUnits": Array [],
    "dynamicsListOfDivision": Array [],
    "dynamicsOtherCompany": Array [],
    "dynamicsOtherContract": Array [],
    "dynamicsOtherSubContractor": Array [],
    "dynamicsSubcontractor": Array [],
    "error": null,
  },
  "dynamicsOtherSubContractor": Array [],
  "dynamicsSubcontractor": Array [],
  "error": null,
}
`;

exports[`Dynamic data reducer test cases should handle DYNAMIC_OTHER_SUB_CONTRACTOR_SUCCESS  1`] = `
Object {
  "dynamicsCompany": Array [],
  "dynamicsContract": Array [],
  "dynamicsListOfBusinessUnits": Array [],
  "dynamicsListOfDivision": Array [],
  "dynamicsOtherCompany": Array [],
  "dynamicsOtherContract": Array [],
  "dynamicsOtherSubContractor": Object {
    "dynamicsCompany": Array [],
    "dynamicsContract": Array [],
    "dynamicsListOfBusinessUnits": Array [],
    "dynamicsListOfDivision": Array [],
    "dynamicsOtherCompany": Array [],
    "dynamicsOtherContract": Array [],
    "dynamicsOtherSubContractor": Array [],
    "dynamicsSubcontractor": Array [],
    "error": null,
  },
  "dynamicsSubcontractor": Array [],
  "error": null,
}
`;

exports[`Dynamic data reducer test cases should handle DYNAMIC_SUB_CONTRACTOR_DATA_GET_ERROR  1`] = `
Object {
  "dynamicsCompany": Array [],
  "dynamicsContract": Array [],
  "dynamicsListOfBusinessUnits": Array [],
  "dynamicsListOfDivision": Array [],
  "dynamicsOtherCompany": Array [],
  "dynamicsOtherContract": Array [],
  "dynamicsOtherSubContractor": Array [],
  "dynamicsSubcontractor": Array [],
  "error": Object {
    "error": true,
  },
}
`;

exports[`Dynamic data reducer test cases should handle DYNAMIC_SUB_CONTRACTOR_DATA_GET_SUCCESS  1`] = `
Object {
  "dynamicsCompany": Array [],
  "dynamicsContract": Array [],
  "dynamicsListOfBusinessUnits": Array [],
  "dynamicsListOfDivision": Array [],
  "dynamicsOtherCompany": Array [],
  "dynamicsOtherContract": Array [],
  "dynamicsOtherSubContractor": Array [],
  "dynamicsSubcontractor": Object {
    "dynamicsCompany": Array [],
    "dynamicsContract": Array [],
    "dynamicsListOfBusinessUnits": Array [],
    "dynamicsListOfDivision": Array [],
    "dynamicsOtherCompany": Array [],
    "dynamicsOtherContract": Array [],
    "dynamicsOtherSubContractor": Array [],
    "dynamicsSubcontractor": Array [],
    "error": null,
  },
  "error": null,
}
`;
